content/applications/hr/appraisals/reporting.rst

```diff
@@ -0,0 +1,188 @@
+
+The report displays the current year, in a default Gantt view, and is grouped by department.
+
+To change the period of time that is presented by default, adjust the date settings in the top-left
+of the report. The options to display are :guilabel:`Day`, :guilabel:`Week`, :guilabel:`Month`, and
+:guilabel:`Year`. Use the arrows to move forward or backward in time.
+
+At any point, click the :guilabel:`Today` button to have the Gantt view include today's date in the
+view.
+
+The report can have other filters and groupings set in the :guilabel:`Search...` bar at the top.
+
+.. image:: reporting/analysis.png
+   :align: center
+   :alt: A report showing all the appraisals for the Appraisal Analysis report.
+
+.. _appraisals/skills-report:
+
+Filter appraisals
+~~~~~~~~~~~~~~~~~
+
+It is possible to only view appraisals with a specific status, by using one of the preconfigured
+filters for the :guilabel:`Appraisal Analysis` report.
+
+To view the available filters, click the :icon:`fa-caret-down` :guilabel:`(caret down)` icon in the
+:guilabel:`Search...` bar.
+
+.. image:: reporting/filter-appraisals.png
+   :align: center
+   :alt: A report showing all the skills grouped by employee.
+
+In the :guilabel:`Filters` section on the left-side of the resulting drop-down menu, three options
+are presented:
+
+- :guilabel:`To Start`: select this filter to **only** present appraisals that have been scheduled
+  (a date populates the :guilabel:`Appraisal Date` field on the appraisal form), but have **not**
+  been confirmed. These appear in gray.
+- :guilabel:`Appraisal Sent`: select this filter to **only** present appraisals that have been
+  scheduled and confirmed, but have **not** been completed. This includes appraisals that have been
+  confirmed but not started, as well as appraisals that are in-process. These appraisals appear in
+  orange.
+- :guilabel:`Done`: select this filter to **only** present appraisals that have been completed and
+  marked as :guilabel:`Done`. These appraisals appear in yellow.
+
+It is possible to select as many or as few filters as desired.
+
+Custom filters
+**************
+
+Appraisals that have been canceled appear in red on the :guilabel:`Appraisal Analysis` report, but
+there is no preconfigured filter to show only cancelled appraisals.
+
+To view only cancelled appraisals, click the :icon:`fa-caret-down` :guilabel:`(caret down)` icon in
+the :guilabel:`Search...` bar.
+
+Next, click :guilabel:`Add Custom Filter` in the :guilabel:`Filters` section, and a :guilabel:`Add
+Custom Filter` pop up window loads.
+
+Using the drop-down menu, select :guilabel:`Status` for the :guilabel:`ID` field, then select
+:guilabel:`Cancelled` for the second drop-down field. Click the :guilabel:`Add` button, and only
+appraisals that have been cancelled appear.
+
+.. image:: reporting/custom-filter.png
+   :align: center
+   :alt: The Custom Filter pop-up with the parameters set to only show cancelled appraisals.
```

Comment:
To streamline this reporting document, and to cut down on re-writing Odoo essentials Reporting information, I suggest reducing this section to a simple example admonition that outlines a perfect use-case for an advanced filter configuration :)
```suggestion
.. example::
   Appraisals that have been cancelled appear in red on the :guilabel:`Appraisal Analysis` report,
   but there is no preconfigured filter to show only cancelled appraisals.

   To view only cancelled appraisals, click the :icon:`fa-caret-down` :guilabel:`(caret down)` icon
   in the :guilabel:`Search...` bar.

   Next, click :guilabel:`Add Custom Filter` in the :guilabel:`Filters` section, and a
   :guilabel:`Add Custom Filter` pop up window loads.

   Using the drop-down menu, select :guilabel:`Status` for the first drop-down, then select
   :guilabel:`Cancelled` for the third drop-down field. Click the :guilabel:`Add` button, and only
   appraisals that have been cancelled appear.

   .. image:: reporting/custom-filter.png
      :align: center
      :alt: The Custom Filter pop-up with the parameters set to only show cancelled appraisals.
```